base/sparse/sparsematrix.jl

```diff
@@ -318,7 +318,218 @@ function sparse_IJ_sorted!{Ti<:Integer}(I::AbstractVector{Ti}, J::AbstractVector
     return SparseMatrixCSC(m, n, colptr, I, V)
 end
 
-## sparse() can take its inputs in unsorted order (the parent method is now in csparse.jl)
+"""
+    sparse(I, J, V, [m, n, combine])
+
+Create a sparse matrix `S` of dimensions `m` x `n` such that `S[I[k], J[k]] = V[k]`. The
+  `combine` function is used to combine duplicates. If `m` and `n` are not specified, they
+  are set to `maximum(I)` and `maximum(J)` respectively. If the `combine` function is not
+  supplied, duplicates are added by default. All elements of `I` must satisfy
+  `1 <= I[k] <= m`, and all elements of `J` must satisfy `1 <= J[k] <= n`.
+
+For additional documentation and an expert driver, see `Base.SparseArrays.sparse!`.
+"""
+function sparse{Tv,Ti<:Integer}(
```

Comment:
That's less standard through the codebase. I prefer body plus one indent for line continuations of function arguments, but a lot of other people seem to like aligning to the paren.